src/gtfs-rt/client.js

```diff
@@ -0,0 +1,72 @@
+import GtfsRealtimeBindings from 'gtfs-realtime-bindings';
+import logger from '../lib/logger.js';
+
+const API_KEY = process.env.GTFS_REALTIME_TOKEN;
+const ALERTS_URL = process.env.GTFS_REALTIME_ALERTS_URL;
+
+let _alertsCache;
+let _alertsCacheTime;
+let _alertsPromise;
+// We're allowed to make 60 requests per hour, let's be conservative and
+// cache for 3min to make it a maximum of 20
+const CACHE_LENGTH = 3 * 60 * 1000;
```

Comment:
can you move this to env env var so we can tweak it as needed without needing to do a new build.

Reply:
done